var/spack/repos/builtin/packages/g4emlow/package.py

```diff
@@ -18,6 +18,7 @@ class G4emlow(Package):
     maintainers("drbenmorgan")
 
     # Only versions relevant to Geant4 releases built by spack are added
+    version("8.5", sha256="c97be73fece5fb4f73c43e11c146b43f651c6991edd0edf8619c9452f8ab1236")
```

Comment:
Can you double check this one?
```
10:41:09 wdconinc@menelaos ~/EIC/juggler (main *$%=) $ spack checksum g4emlow 8.5
==> Found 1 version of g4emlow
==> Fetching https://geant4-data.web.cern.ch/geant4-data/datasets/G4EMLOW.8.5.tar.gz

    version("8.5", sha256="66baca49ac5d45e2ac10c125b4fb266225e511803e66981909ce9cd3e9bcef73")

10:42:27 wdconinc@menelaos ~/EIC/juggler (main *$%=) $ spack checksum g4emlow 6.50
==> Found 1 version of g4emlow
==> Fetching https://geant4-data.web.cern.ch/geant4-data/datasets/G4EMLOW.6.50.tar.gz

    version("6.50", sha256="c97be73fece5fb4f73c43e11c146b43f651c6991edd0edf8619c9452f8ab1236")

```
so I think:
```suggestion
    version("8.5", sha256="66baca49ac5d45e2ac10c125b4fb266225e511803e66981909ce9cd3e9bcef73")
```

Reply:
Not sure what's going on here: I removed the cache and reinstalled and the existing checksum worked:
```
$ spack install --overwrite g4emlow@8.5
==> The following package specs will be reinstalled:

-- darwin-sonoma-m2 / apple-clang@15.0.0 ------------------------
f6g3sno g4emlow@8.5%apple-clang  build_system=generic
==> Do you want to proceed? [y/N] y
==> Installing g4emlow-8.5-f6g3sno2c5ga7yjbxaejw7u4yhz4eksp [1/1]
==> No binary for g4emlow-8.5-f6g3sno2c5ga7yjbxaejw7u4yhz4eksp found: installing from source
==> Fetching https://mirror.spack.io/_source-cache/archive/c9/c97be73fece5fb4f73c43e11c146b43f651c6991edd0edf8619c9452f8ab1236.tar.gz
==> No patches needed for g4emlow
==> g4emlow: Executing phase: 'install'
==> g4emlow: Successfully installed g4emlow-8.5-f6g3sno2c5ga7yjbxaejw7u4yhz4eksp
  Stage: 2.26s.  Install: 1.94s.  Post-install: 0.94s.  Total: 5.16s
[+] /opt/spack/opt/spack/sonoma/g4emlow/8.5/f6g3sno
```
but maybe I broke something? The hash you give definitely works (and pulls from the original URL rather than the spack cache) so I'll just use that.